### DISCUSSION TITLE
QQ 群解散记录

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#86] [杂谈]
![5efd17adefc2b80d6339096f3ae943e6_720](https://github.com/user-attachments/assets/7a7edc71-52ad-41a4-afd2-9c307dc905ca)
说@全体人员还有次数限制，要充钱开会员才可以，导致有三四个群只是留言，没有通知到全体成员，明天再看看。糊了！

**Comment by liu0852967** [comment]
妈耶，舍不得，呜呜呜

**Comment by Mayleaves** [comment]
😭已经用了近两年，好幸运能遇到这个软件。作者也好好，不仅会答复群问题，还会添加新功能。为爱发电太感动了🌹
（好舍不得……

**Comment by WatsonYao** [comment]
![Screenshot_2024-09-28-08-08-35-036_com tencent mobileqq-edit](https://github.com/user-attachments/assets/abdfaaa7-c49e-4059-985f-e9acbdf777ca)

**Comment by WatsonYao** [comment]
其他平台的也都解散了，只保留了 Github 上的这个讨论组。

**Reply by shy2642006935** [reply]
> 其他平台的也都解散了，只保留了Github上的这个讨论组。

作者大大能加个qq吗，一直用您的软件很久了

**Reply by WatsonYao** [reply]
有反馈或建议可以发这里或邮件吧
